internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -748,8 +769,9 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context,
 		if clusterInstall.Spec.IgnitionEndpoint.Url != "" {
 			ignitionEndpoint.URL = swag.String(clusterInstall.Spec.IgnitionEndpoint.Url)
 		}
-		if clusterInstall.Spec.IgnitionEndpoint.CaCertificate != "" {
-			ignitionEndpoint.CaCertificate = swag.String(clusterInstall.Spec.IgnitionEndpoint.CaCertificate)
+		caCertificate, err := r.getEncodedCACert(ctx, log, clusterDeployment, clusterInstall)
+		if err == nil {
```

Comment:
We should not ignore errors. How would we know something is wrong?

Reply:
Fixed this